src/provider/element-templates/ElementTemplates.js

```diff
@@ -194,8 +209,90 @@ export default class ElementTemplates {
 
     this._commandStack.execute('propertiesPanel.camunda.changeTemplate', context);
 
+    this._eventBus.fire(`elementTemplates.${action}`, payload);
+
     return context.element;
   }
+
+  /**
+   * Remove template from a given element.
+   *
+   * @param {djs.model.Base} element
+   */
+  removeTemplate(element) {
+    const replace = this._injector.get('replace'),
+          selection = this._injector.get('selection'),
+          eventBus = this._injector.get('eventBus');
+
+    eventBus.fire('elementTemplates.remove', { element });
+
+    const businessObject = getBusinessObject(element);
+
+    const type = businessObject.$type,
+          eventDefinitionType = getEventDefinitionType(businessObject);
+
+    const newBusinessObject = createBlankBusinessObject(element, this._injector);
+
+    const newElement = replace.replaceElement(element, {
+      type: type,
+      businessObject: newBusinessObject,
+      eventDefinitionType: eventDefinitionType
+    });
+
+    selection.select(newElement);
```

Comment:
Could you verify if this is necessary? We handle selection after replace in diagram-js now, cf. https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/replace/ReplaceSelectionBehavior.js#L3.

Reply:
Yup, we don't need this. I removed the selection here